<launch>
    <!-- Common arguments -->
    <arg name="launch_nuc" default="nuc1"/>
    

    <!-- NUC 1 -->
    <group if="$(eval launch_nuc == 'nuc1')">
        <include file="$(find aruco_detect)/launch/sine_ft.launch">
        <arg name="nuc_name" value="nuc1"/>
        <arg name="amplitude" default="1"/>
        <arg name="frequency" default="1"/>
        <arg name="fiducial_id" default="1"/>
        </include>
    </group>

    <!-- NUC 2 -->
    <group if="$(eval launch_nuc == 'nuc2')">
        <include file="$(find aruco_detect)/launch/sine_ft.launch">
        <arg name="nuc_name" value="nuc2"/>
        <arg name="amplitude" default="2"/>
        <arg name="frequency" default="2"/>
        <arg name="fiducial_id" default="2"/>
        </include>
    </group>


    <!-- NUC 3 -->
    <group if="$(eval launch_nuc == 'nuc3')">
        <include file="$(find aruco_detect)/launch/sine_ft.launch">
        <arg name="nuc_name" value="nuc3"/>
        <arg name="amplitude" default="3"/>
        <arg name="frequency" default="3"/>
        <arg name="fiducial_id" default="3"/>
        </include>
    </group>

</launch>